Dockerfile

```diff
@@ -1,9 +1,6 @@
-# Building on Ubuntu Bionic (18.04) and supporting glibc 2.27. This allows
```

Comment:
Does this comment no longer apply?

Reply:
ebpf_exporter is built by container, so I'd think ebpf_exporter be better to run in a container,  it is no need to care about the os/libc version on the host.

Reply:
Everything is built in a container these days, because it's convenient and somewhat reproducible. We only distribute a binary, though, not a docker image. I would like to keep the binary with max compatibility, as people seem to depend on it: #115.

For people who don't run Docker, it's certainly not an option to install it just for ebpf_exporter.

Reply:
@bobrik : could you give it a try with the script in https://github.com/cloudflare/ebpf_exporter/pull/130#issuecomment-1046722298 ?